src/plugins/pbjx/reducers/index.js

```diff
@@ -39,7 +39,19 @@ const onResponseCleared = (prevState = {}, action) => {
   return state;
 };
 
+const onUpdateResponseNode = (prevState = {}, { node, channel, curie }) => {
+  const state = { ...prevState };
+  const curieStr = `${curie}`;
+  const newResponse = state[channel][curieStr].response.clone();
+
+  newResponse.set('node', node);
```

Comment:
@rsumilang this is pretty weird man. now we are hijacking the pbjx state to force the form to change? doesnt feel right

Reply:
@efjacobson yes, now it's not as sensitive to forms changing and having to keep an eye on new form components. changing the pbjx state allows for each subscriber to do its thing.